core/src/main/java/io/questdb/cairo/security/AllowAllSecurityContext.java

```diff
@@ -173,4 +173,9 @@ public void checkEntityEnabled() {
     public String getPrincipal() {
         return Constants.USER_NAME;
     }
+
+    @Override
+    public boolean isSystemAdmin() {
```

Comment:
we could have a default value on the interface for this

Reply:
Thought about that, but the sensible default implementation would be to return `true`, since most implementations do that.
However, this is not safe to do, because if a new SecurityContext implementation is added to the codebase, it automatically grants system admin to everyone. By forcing the developer to implement the method, they have to think about it.
We have removed defaults from SecurityContext in the past because of this.